src/transformers/modeling_longformer.py

```diff
@@ -0,0 +1,667 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch Longformer model. """
+
+
+import logging
+import math
+from functools import lru_cache
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+from torch.nn import CrossEntropyLoss
+
+from .configuration_longformer import LongformerConfig
+from .file_utils import add_start_docstrings, add_start_docstrings_to_callable
+from .modeling_roberta import RobertaModel, RobertaLMHead
+from .modeling_bert import BertPreTrainedModel
+
+
+logger = logging.getLogger(__name__)
+
+# TODO: upload model and config files
+LONGFORMER_PRETRAINED_MODEL_ARCHIVE_MAP = {
+    "longformer-base-4096": "https://ai2-s2-research.s3-us-west-2.amazonaws.com/longformer/longformer-base-4096/pytorch_model.bin",
```

Comment:
I can take care of uploading the model to the HF model page :-)

Reply:
Actually, I think you guys already have an organization `allenai` - so I think you can upload the models to `allenai/longformer-base-4096` and `allenai/longformer-large-4096`. Since we have "organizations" now for models we can directly link it to them :-)